nekoyume/Assets/_Scripts/BlockChain/ActionRenderHandler.cs

```diff
@@ -663,6 +663,13 @@ private void ResponseHackAndSlash(ActionBase.ActionEvaluation<HackAndSlash> eval
         {
             if (eval.Exception is null)
             {
+                if (eval.Action.Id != Game.Game.instance.ActionManager.LastBattleActionId ||
+                    eval.Action.worldId != Game.Game.instance.ActionManager.LastBattleWorldId ||
+                    eval.Action.stageId != Game.Game.instance.ActionManager.LastBattleStageId)
```

Comment:
```cs
if (eval.Action.Id != Game.Game.instance.ActionManager.LastBattleActionId ||
    eval.Action.worldId != Game.Game.instance.ActionManager.LastBattleWorldId ||
    eval.Action.stageId != Game.Game.instance.ActionManager.LastBattleStageId)
```
This statement overlaps with if statement of `ResponseMimisbrunnr` method.
Why don't you make a method to check this?

Reply:
OK I'll try.